docs/referenceConf.js

```diff
@@ -206,7 +210,7 @@ exports.config = {
     //
     // If you need access back to the current configuration object,
     // use a pattern like the following:
-    //     browser.getProcessedConfig().then(function(config) {
+    //     return browser.getProcessedConfig().then(function(config) {
```

Comment:
Shouldn't this be the default? It indicates that you can return and it will be used for something.

It seems like a better idea than not returning anything here and having the user scratch their head, spend a long time googling, searching docs and reading source code. That's exactly why I came up with this PR.

Besides, leaving a loose promise more often than not is not what you want.